packages/dataviews/src/dataviews.js

```diff
@@ -24,7 +24,7 @@ export default function DataViews( {
 	searchLabel = undefined,
 	actions,
 	data,
-	getItemId,
+	getItemId = ( item ) => item.id,
```

Comment:
Maybe we should declare it as `const` outside the component, since it's a function?

Reply:
👍 [6cc4858](https://github.com/WordPress/gutenberg/pull/57308/commits/6cc485826f06b2f17dcbc391a38b226a060745da)